bot/cogs/tags.py

```diff
@@ -92,7 +92,18 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
         """Show all known tags, a single tag, or run a subcommand."""
         await ctx.invoke(self.get_command, tag_name=tag_name)
 
-    async def _get_command(self, ctx: Context, tag_name: TagNameConverter = None) -> None:
+    async def get_command(self, ctx: Context, tag_name: str = None) -> bool:
+        """
+        Shows a specific tag or a suggestion within a given context.
+
+        params:
+            ctx - a Context object , being used to send and choose the right tag
+            tag_name - a string representing the searched tag name
+
+        Returns False if no tag is not found or the tag is on cooldown
+
+        Return True if found and sent a tag
```

Comment:
Keeping in mind this is not strictly enforced, here are my suggestions for writing docstrings:

1. Use the imperative mood.
2. Try to include information about the return value in the summary, if it fits.
3. A parameters/arguments section is generally something we don't use. If they warrant explicit explanation, then that information should be provided as sentences/paragraphs in the body of the docstring.

My attempt, feel free to modify:

```py
        """
        Show contents of the tag `tag_name` in `ctx` and return True if something is shown.

        If a tag is not found, display similar tag names as suggestions. If a tag is not specified,
        display a paginated embed of all tags.

        Tags are on cooldowns on a per-tag, per-channel basis. If a tag is on cooldown, display
        nothing and return False.
        """
```